.github/workflows/build-packages.yml

```diff
@@ -2,7 +2,7 @@ name: "Debian Package Builder"
 
 on:
   push:
-    branches: [ main ]
+    branches: [ main, wfdhaka_online ]
   pull_request:
```

Comment:
```suggestion
```
I think delete this line too so it only builds on the main branch, not when someone makes a PR

Reply:
Done